app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    instances_list = []
+    for person in people:
+        instances_list.append(Person(person["name"], person["age"]))
```

Comment:
I don't see list comprehension :(
```suggestion
    instances_list = [
        Person(person["name"], person["age"]) for person in people
    ]
```

Reply:
@OleksandrPozden  Thank you